scalafmt-tests/src/test/resources/newlines/source_keep.stat

```diff
@@ -1327,7 +1328,8 @@ object WrapperToHaveStatTestCaseParserWorking {
       someInt = 5
     ) @annot3
     @deprecated(value =
-      "bar") def bar2 = 1
+      "bar"
```

Comment:
break here is needed because of dangling parentheses setting, and consistent with the expression just above.